deploy/templates/bicep/UploadTemplates.bicep

```diff
@@ -0,0 +1,111 @@
+param baseName string 
+param location string 
+param resourceIdentityResolutionType string 
+
+resource storageAccount 'Microsoft.Storage/storageAccounts@2022-09-01' existing = {
+  name: '${baseName}sa'
+}
+
+resource userAssignedMI 'Microsoft.ManagedIdentity/userAssignedIdentities@2023-01-31' existing = {
+  name: '${baseName}UAMI'
+}
+
+var storageBlobDataOwnerRoleId = resourceId('Microsoft.Authorization/roleDefinitions', 'b7e6dc6d-f1e8-4753-8033-0f276bb0955b')
+
+resource storageBlobDataOwnerDeployment 'Microsoft.Authorization/roleAssignments@2022-04-01' = {
+  scope: storageAccount
+  name: guid(storageBlobDataOwnerRoleId, userAssignedMI.id, storageAccount.id)
+  properties: {
+    roleDefinitionId: storageBlobDataOwnerRoleId
+    principalId: userAssignedMI.properties.principalId
+    principalType: 'ServicePrincipal'
+  }
+}
+
+resource deploymentStorageAccount 'Microsoft.Storage/storageAccounts@2022-09-01' = {
+  name: '${baseName}deploysa'
+  location: location
+  tags: {
+    IomtFhirConnector: 'ResourceIdentity:${resourceIdentityResolutionType}'
+    IomtFhirVersion: 'R4'
+  }
+  kind: 'StorageV2'
+  sku: {
+    name: 'Standard_RAGRS'
+  }
+}
+
+param blobContainerName string = 'template'
+param devicecontentFile string = 'devicecontent.json'
+param fhirmappingFile string = 'fhirmapping.json'
+param devicecontentURL string = 'https://raw.githubusercontent.com/microsoft/iomt-fhir/main/sample/templates/basic/devicecontent.json'
+param fhirmappingURL string = 'https://raw.githubusercontent.com/microsoft/iomt-fhir/main/sample/templates/basic/devicecontent.json'
```

Comment:
Typo, should be FHIR mapping. Also double check the content.  I don't believe the device mapping has patient id and won't work with create mode.  Feel free to update that template.  Please double check with the content of the templates you deleted to make sure nothing else is missed.

Reply:
Thanks for catching this! You are right, the device mapping did not include patient id. The template has been updated to include patientid.